cmd/maya-exporter/app/command/exporter.go

```diff
@@ -31,10 +34,25 @@ func Initialize(options *VolumeExporterOptions) string {
 // info.
 
 // StartMayaExporter starts an HTTP server that exposes the metrics on
-// "/metrics" endpoint.
+// <clusterIP>:9500/metrics endpoint and for getting the metrics in json
+// the "type=json" action can be used. e.g <clusterIP>:9500/metrics/?type=json
 func (options *VolumeExporterOptions) StartMayaExporter() error {
 	glog.Info("Starting http server....")
 	http.Handle(options.MetricsPath, promhttp.Handler())
+	http.HandleFunc(options.MetricsPath+"/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
the queries should be of the form `/metrics?type=json`

Reply:
For this, we can't use the handler provided by the vendor. We have to make out own custom handler function .

Reply:
and I don't thik using rest?type=json will be a good idea. Here is nice example of rest api that uses similar pattern to our usecase https://clist.by/api/v1/?format=json. Rather we can use /metrics/json/

Reply:
and i think we should the endpoints should end with a `/`. A very good answer is present here regarding this https://softwareengineering.stackexchange.com/questions/186959/trailing-slash-in-restful-api .